### DISCUSSION TITLE
Amiga 500 Kickstart not found

### DISCUSSION THREAD

**Discussion by Santorayo3** [eduardofilo/RG350_adam_image#136] [General]
Hello,
I get the "object not found kick34005.a500" error message when I try to load an Amiga 500 game.
I've put the correct (verified MD5) rom in the bios directory.
Other Amiga versions just work fine but the 500 rom is somehow not recognized correctly.

Any ideas?

**Comment by eduardofilo** [comment]
Hello.
I don't think that's it, because on the EXT card, the FAT32 file system is not case sensitive, but according to the core documentation puae_libretro, the BIOS name should be `kick34005.A500` (with capital A). Try that and if it doesn't work tonight I test in my machine to see if I find any other problems.

**Comment by Santorayo3** [comment]
Thanks,
I've tried that but the ending was already in upper case. Curiously the error message stated that kick34005.a500 with a lower case was not found. So I tried to rename it to lower case... just in case *badum-tss* but that didn't workout either.

**Reply by eduardofilo** [reply]
Double check the BIOS again. I have tested on my unit, and removing the file from the bios directory at the root of SD EXT, I get the warning:

![screenshot003](https://user-images.githubusercontent.com/2923728/147835251-29be982d-7942-4e38-b5dc-a636e77142cc.png)

But when I restore it the same demo that I had tried before no longer shows it:

![screenshot004](https://user-images.githubusercontent.com/2923728/147835271-6847e652-aee8-4234-8e0f-cf5fc7c95fd3.png)

```
rg280v:/media/sdcard/bios $ md5sum kick34005.A500 
82a21c1890cae844b3df741f2762d48d  kick34005.A500
```

**Comment by Santorayo3** [comment]
I don't get the error message from the RetroArch but from ermm "inside" the game? Right after the WDHL Loader.
See this photo: https://ibb.co/SdbpHYz

**Reply by eduardofilo** [reply]
I'm afraid I don't know enough about this system. I can only get that RA reads the BIOS correctly. Sorry. Let's see if someone with a more knowledge can help you.
I have been trying to read a bit and it seems that the WHDL system acts as an abstraction layer between the different versions of the Amiga platform. Then you will probably need to locate the different BIOSes in another directory. In fact, in the photo you have uploaded, it indicates the path `kickstarts/kick34005.a500`. Try creating that path next to the ROM you are trying to run.
By the way, what is ermm?

**Comment by Santorayo3** [comment]
"Ermmm" was just an expression of thinking :D

Did some more research. I had the Amiga Model set to "Auto". But afaik that game was very old and might not run on anything but the A500. So I selected the A500 model. I can boot that up without loading a rom - so the Kickstarter seems fine - but when I try to load the rom I get another error message "Unknown Command Fail 41" and not even google can tell my anything about that. 😅

As it seems I can't run any roms with the A500 Model.

**Reply by eduardofilo** [reply]
> "Ermmm" was just an expression of thinking :D

XD

> Did some more research. I had the Amiga Model set to "Auto". But afaik that game was very old and might not run on anything but the A500. So I selected the A500 model. I can boot that up without loading a rom - so the Kickstarter seems fine - but when I try to load the rom I get another error message "Unknown Command Fail 41" and not even google can tell my anything about that. sweat_smile
> 
> As it seems I can't run any roms with the A500 Model.

In case it helps, I adopted the configuration suggested in issue #17 for the Amiga system.

**Comment by Santorayo3** [comment]
I have fixed the issue! 🥳
Changing the "WHDLoad Support" in the RetroArch Options from "files" to "hdfs" solved it. :)

**Reply by eduardofilo** [reply]
Great!
Do you think it is safe to adopt that option in general?

**Reply by Santorayo3** [reply]
I don't know... but I didn't run into any issues with it. I've started all 30 or so games I have and they all booted up fine.

**Reply by eduardofilo** [reply]
OK. So I think I will adopt the setting for the image. Thank you.
